spike on rewriting the model

jameswhite opened the pull request:
I didn't like the old one after noodling about id for a while...

Commit:
spike on rewriting the model

Commit:
some more thoughts

Commit:
more thoughts

Commit:
we're going to need a database

Commit:
use a database instead

azizshamim commented on README.md:
```diff
@@ -0,0 +1,83 @@
+  - If a Human costs you 10,000 paper, that is deducted every turn
+  - If an Implementation has an upkeep of 100, it is deducted every turn.
+  - If an Implementation has an upkeep of 100, it is deducted every turn.
+  - 200 Turns can be considered roughly a year in game time.
```
That's a LOT of turns. How long is this game designed to last?

jameswhite replied:
It depends on the goal you set. and a turn is just a "clock step" like 60 seconds in real time. The idea is to give direction and the "hires" will work autonomously until they get hung up on what to work on next. we could make it fewer turns, I picked 200 because that's 2000/10 or the number of hours in a year/10

Commit:
rails and stuff

Commit:
try the version of openlayers the last one recommended

stephenyeargin committed:
Level up the .gitignore

This steals from https://github.com/github/gitignore/blob/master/Rails.gitignore

stephenyeargin committed:
Merge pull request #2 from stephenyeargin/gitignore

Level up the .gitignore
